vue/src/vueTextMask.js

```diff
@@ -1,23 +1,91 @@
 import createTextMaskInputElement from '../../core/src/createTextMaskInputElement'
 
 export default {
-  textMaskInputElement: null,
+  template: `
+    <input ref="input"
+      :value="value"
+      @input="updateValue($event.target.value)"
+    />
+  `,
 
-  inputHandler({target: {value}}) {
-    return this.textMaskInputElement.update(value)
+  name: 'masked-input',
+
+  props: {
+    value: {
+      type: String,
+      required: false,
+      default: ''
+    },
+
+    mask: {
+      type: [Array, Function],
+      required: true
```

Comment:
@lozjackson According to the [text mask documentation](https://github.com/text-mask/text-mask/blob/master/componentDocumentation.md#mask) the `mask` property can either be an array or a function. The [_function_](https://github.com/text-mask/text-mask/blob/master/componentDocumentation.md#mask-function) can return false, but the mask itself cannot be false.

I have tested returning false from both: 
(1) the pipe function
(2) the mask function

Both work as intended